dev-libs/libsodium/libsodium-1.0.16.ebuild

```diff
@@ -37,7 +40,10 @@ src_configure() {
 		${myconf}
 }
 
-src_install() {
-	default
-	prune_libtool_files
+multilib_src_install() {
```

Comment:
You can remove `multilib_src_install` since the default eclass implementation will just work™.

Reply:
I will remove it.